etc/fluentbit/fluentbit.conf

```diff
@@ -0,0 +1,21 @@
+[INPUT]
+    Name        tcp
+    Listen      localhost
+    Port        5170
+    Format      json
+
+[FILTER]
+    Name    lua
+    Match   *
+    script  /home/vcap/app/.local/fluentbit/redaction.lua
```

Comment:
Shouldn't `redaction.lua` suffice as the lookup is relative to the config file ?